app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/disk/AuthDiskSource.kt

```diff
@@ -50,6 +50,8 @@ interface AuthDiskSource {
      * Clears all the settings data for the given user.
      *
      * Note that this does not include any data saved in the [userState].
+     * This also does not include values saved via [storeOnboardingStatus] this
+     * is considered a "per install" value and as such should not be removed on logout.
```

Comment:
I don't think the caller needs to know this, the spot in the implementation is probably enough